### DISCUSSION TITLE
How to define for dependencies in the Nuget package (ControlzEx)

### DISCUSSION THREAD

**Discussion by ghost** [MahApps/MahApps.Metro#4171] [Q&A] [answered]
Hi, just a small question. But the [Nuget package](https://www.nuget.org/packages/MahApps.Metro/) requires "ControlzEx (>= 4.4.0)".

This causes that when you include MahApps.Metro via PackageReference it uses an old version of ControlzEx. Is there any way to fix this without including ControlzEx explicitly as a PackageReference?

_You can also see this very well if you create a project with the new csproj-style and include MahApps.Metro (via nuget). Then you can see the dependencies it uses under references._

**Comment by punker76** [comment]
@Rerago It seems like that this is currently not available for PackageReference out of the box.

- https://stackoverflow.com/q/60868895/920384
- https://github.com/NuGet/Home/issues/5553

**Comment by ghost** [comment]
I know that the I can not currently set version itself and/or update. But the question is rather why the nuget shows 4.4.0 although [here](https://github.com/MahApps/MahApps.Metro/blob/develop/src/MahApps.Metro/MahApps.Metro.csproj#L19) explicitly a v5 is indicated.

**Comment by punker76** [comment, marked as answer]
@Rerago Because you are look to the dev branch and not the release branch [here](https://github.com/MahApps/MahApps.Metro/blob/ee63390e64f4688ca049bafb8a0c74f224a4ecb9/src/MahApps.Metro/MahApps.Metro.csproj#L18).

**Comment by ghost** [comment]
Ahhhhhhh ok. Then my mistake. Means the next update will also use the latest ControlzEx. Then I am satisfied so far :)
